patches/ygot/ygot.patch

```diff
@@ -1,6 +1,58 @@
+diff -ruN ygot-dir-orig/ygot/generator/generator.go ygot-dir/ygot/generator/generator.go
+--- ygot-dir-orig/ygot/generator/generator.go	2020-06-21 15:35:13.777667000 -0700
++++ ygot-dir/ygot/generator/generator.go	2020-06-21 15:27:34.931674000 -0700
+@@ -109,7 +109,15 @@
+ }
+ 
+ // writeIfNotEmpty writes the string s to b if it has a non-zero length.
+-func writeIfNotEmpty(b io.StringWriter, s string) {
++// commented out the below method, since "io.StringWriter" is introduced in go 1.12
++//func writeIfNotEmpty(b io.StringWriter, s string) {
```

Comment:
Please remove functions that are not in use. We could bring it back, when it is required in future.